src/zedwallet++/ParseArguments.h

```diff
@@ -28,7 +28,7 @@ struct ZedConfig
     std::string walletPass;
 
     /* Controls what level of messages to log */
-    Logger::LogLevel logLevel = Logger::DISABLED;
+    Logger::LogLevel logLevel = Logger::FATAL;
```

Comment:
You sure you want to default to debug logging? Makes the wallet quite tricky to use..

Reply:
This one was most likely a forgotten relict from previous debugging. Changing back to disabled.